content/4_Gold/Tree_Euler.mdx

```diff
@@ -542,6 +563,89 @@ public class Main {
 
 </JavaSection>
 
+<PySection>
+
+```py
+import sys
+
+sys.setrecursionlimit(10**9)
```

Comment:
```suggestion
sys.setrecursionlimit(10 ** 9)
```

Reply:
Done.

Reply:
no it isn't

Reply:
I think the `pre-commit.ci` bot  keeps changing the file.